tests/test_logging.py

```diff
@@ -64,34 +63,39 @@ def test_default_logger_levels_and_split(capsys):
 
 
 @pytest.mark.skipif(not HAVE_OPM, reason="Command line client requires OPM")
-@pytest.mark.parametrize(
-    "verbose, fileexport",
-    itertools.product([True, False], [True, False]),
-)
-def test_pyscal_logging(tmp_path, verbose, fileexport, mocker, capsys):
-    """Test that the command line client for each submodule logs correctly.
-    Each submodule should write logs to stdout for INFO and WARNING messages
-    when they write to dedicated output files, but must write to stderr when
-    stdout is used as the output stream. This requres correct configuration
-    in each submodule and must therefore be tested.
-    This test function is more robust if each main() invocation is run in a
-    subprocess, but that also makes it 10 times slower. When not run in a
-    subprocess, the verbosity option must be False before True for the tests to
-    work, this is related (?) to loggers not being properly reset between each
-    test invocation.
+def test_pyscal_logging_verbose(tmp_path, mocker, capsys):
+    """Test that the command line client logs correctly with output set to
+    stdout and verbose set to true.
     """
 
     testdir = Path(__file__).absolute().parent
     relperm_file = testdir / "data/relperm-input-example.xlsx"
     commands = ["pyscal", str(relperm_file), "--output"]
+    commands.append("-")
 
-    if fileexport:
-        commands.append(str(tmp_path / "output.inc"))
-    else:
-        commands.append("-")
+    commands.append("-v")
 
-    if verbose:
-        commands.append("-v")
+    mocker.patch("sys.argv", commands)
+
+    pyscalcli.main()
+    captured = capsys.readouterr()
+    stdout_output = captured.out
+    stderr_output = captured.err
+
+    assert "INFO:" in stderr_output
+    assert "INFO:" not in stdout_output
+
+
+@pytest.mark.skipif(not HAVE_OPM, reason="Command line client requires OPM")
```

Comment:
OPM is not connected that tightly to pyscal, so everything with respect to OPM should be removed.

Reply:
Ok, i have just fixed it.